### PR TITLE
feat(ui): Keyboard shortcuts

### LINKED ISSUE #5466: Keyboard navigation for UI
# Summary

Easy navigation of UI with the keyboard. 

# Motivation

Better accessibility, and productivity with keyboard shortcuts.

# Proposal

### Application List: 
- Navigate applications with arrow keys
- `/` to search apps
- `Enter` to go to selected app

### Application Details
- Navigate resource tree or pod view with arrow keys
- `Enter` to open details panel for selected resource
- `l` key to open logs for selected pod
- `d` key to open app details panel
- `t` key to open app diff
- `ctrl+R` to refresh application

There are many others that can/should be implemented, the above is a first pass - and community suggestions for keybindings are encouraged

### PR DESCRIPTION
**Pull request by rbreeze**
Fixes https://github.com/argoproj/argo-cd/issues/5466

Shortcuts: 
- Arrow key navigation for Applications List (both tile and table views)
- `/` key to search

Also moves/redesigns search bar: 
Screen Shot 2021-03-02 at 10 43 42 AM

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [x] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/application-create-panel/application-create-panel.tsx [hunk 1]
```diff
@@ -231,13 +233,24 @@ export const ApplicationCreatePanel = (props: {
...
                                                 <div className='row argo-form-row'>
                                                     <div className='columns small-10'>
-                                                        <FormField
-                                                            formApi={api}
-                                                            label='Repository URL'
-                                                            qeId='application-create-field-repository-url'
-                                                            field='spec.source.repoURL'
-                                                            component={AutocompleteField}
-                                                            componentProps={{items: repos}}
+                                                        {false && (
```
Comment (alexmt) [inline ui/src/app/applications/components/application-create-panel/application-create-panel.tsx:236]: This `FormField` is not rendered due to `{false && (`, correct? Remove it please

**Review by alexmt — APPROVED** [review]
Awesome! Added reminder to remove the unused component. 
LGTM After resolving it.

File: ui/src/app/applications/components/application-create-panel/application-create-panel.tsx (hunk 1 above)
Comment (rbreeze) [reply ui/src/app/applications/components/application-create-panel/application-create-panel.tsx:236]: I think this was to silence a compiler error while I was debugging, forgot to remove it. Thank you for catching it!

**Comment by rbreeze** [comment]
Addendum #6859

**Pull request merged**